### DISCUSSION TITLE
[FINALIZADO] 🎅Presentão de Natal - 3 cursos ninjas do Daciuk 💙

### DISCUSSION THREAD

**Discussion by felipefialho** [frontendbr/forum#1310] [Dicas]
Fala amigs, como estão? 😄

O fim de ano está se aproximando e vamos dar um presentão de natal: 

- [3 cursos ninjas do Fernando Daciuk](https://www.udemy.com/user/fernando-daciuk/)!  

Basta responder essa issue escolhendo entre:

- JavaScript Ninja
- React Ninja
- Git e Github

No dia 21/12, vamos sortear as três pessoas ganhadoras.

(o sorteio não será por curso, e sim por pessoa, que vai ganhar o curso escolhido)

Sensacional né?

Está valendo e feliz natal pra todos nós! ❤️

**Comment by EdipoYuri** [comment]
Só responder aqui? Então eu escolho o de React Ninja. Presenteia nois :D

**Comment by evertonstrack** [comment]
React Ninja. :)

**Comment by WalterPaes** [comment]
JavaScript Ninja

**Comment by brunocrpontes** [comment]
JavaScript Ninja

**Comment by biavilasb** [comment]
React Ninja 😄

**Comment by lciceris** [comment]
React Ninja

**Comment by DavyGuedes** [comment]
React Ninja

**Comment by MarciaCogitare** [comment]
React ninja

**Comment by n3pu** [comment]
React Ninja

**Comment by paulobrun0** [comment]
JavaScript Ninja

**Comment by andrey-leal** [comment]
Javascript Ninja

**Comment by xavierleonardo** [comment]
Raect Ninja

**Comment by lucianosz7** [comment]
React Ninja

**Comment by lucas-frois** [comment]
React ninja.

**Comment by brennowilliam** [comment]
React Ninja.

**Comment by has256** [comment]
Ótima iniciativa! Feliz natal :tada:.

React Ninja.

**Comment by danieldhsd** [comment]
JavaScript Ninja

**Comment by ClaudioFuriniJr** [comment]
React Ninja

**Comment by elfiservice** [comment]
React Ninja

**Comment by ghost** [comment]
JavaScript Ninja

**Comment by ghost** [comment]
Git e Github

**Comment by sheilagabriela** [comment]
Javascript Ninja

**Comment by BradHick** [comment]
React Ninja

**Comment by RaphaelYuryOliveira** [comment]
JavaScript Ninja

**Comment by mtargino** [comment]
JavaScript Ninja

**Comment by victornery** [comment]
React Ninja. Melhor até que um Nanodegree.

**Comment by juaopedrosilva** [comment]
React Ninja

**Comment by Wellers0n** [comment]
JavaScript Ninja

**Comment by Adelblande** [comment]
JavaScript Ninja

**Comment by KaiqueMunhoz** [comment]
React Ninja

**Comment by chicojunior** [comment]
javascript Ninja

**Comment by gellys** [comment]
React Ninja

**Comment by Kamilahsantos** [comment]
React ninja ❤❤❤

**Comment by kvnol** [comment]
React Ninja ❤️

**Comment by palharez** [comment]
React Ninja!!

**Comment by danilomilan** [comment]
React Ninja

**Comment by ruanvalente** [comment]
React Ninja 💙

**Comment by daniellesantana** [comment]
React Ninja <3

**Comment by onlurking** [comment]
React Ninja!

**Comment by lemellendez** [comment]
Oiii,

Eu quero o Curso de REACT por favor!!

**Comment by brnms** [comment]
Git e Github

**Comment by karluslima** [comment]
React Ninja ( :

**Comment by CristopherJonas** [comment]
Git e Github

**Comment by luketevl** [comment]
React Ninja

**Comment by daniel-maker** [comment]
React Ninja :)

**Comment by anderson-silva-macedo** [comment]
Javascript ninja!

**Comment by gcardoso23** [comment]
React Ninja

**Comment by luizbills** [comment]
React Ninja

**Comment by lucas-barbosa** [comment]
React Ninja

**Comment by milaniromulo** [comment]
React Ninja

**Comment by mlknservices** [comment]
JavaScript Ninja

**Comment by igorhideki** [comment]
JavaScript Ninja

**Comment by thyagorafaell** [comment]
JavaScript Ninja

**Comment by kiq7** [comment]
React Ninja

**Comment by claudiomborges** [comment]
**React Ninja**

**Comment by hebertjulio** [comment]
React Ninja

**Comment by tadeubrasil** [comment]
React Ninja

**Comment by cicerohen** [comment]
JavaScript Ninja

**Comment by pissulin** [comment]
JavaScript Ninja

**Comment by dfirmino** [comment]
React ninja

**Comment by erickmajor** [comment]
React Ninja

**Comment by rubaoma** [comment]
JavaScript Ninja

**Comment by gledsonlaraujo** [comment]
JavaScript Ninja

**Comment by nathanssantos** [comment]
React Ninja

**Comment by joaopjt** [comment]
React Ninja

**Comment by mntoledo** [comment]
React Ninja

**Comment by GuiRamos7** [comment]
React Ninja

**Comment by eldertec** [comment]
React Ninja

**Comment by pcesarteixeira** [comment]
React Ninja

**Comment by hmilena** [comment]
React Ninja

**Comment by thayllachristine** [comment]
React Ninja :)

**Comment by JulioPaiva** [comment]
React Ninjaaaaaaaaaa ::)

**Comment by leomendesm** [comment]
JavaScript Ninja 🥇

**Comment by SandroSena** [comment]
React Ninja

**Comment by EmanuelCampos** [comment]
JavaScript Ninja

**Comment by arturpsl** [comment]
react ninja

**Comment by dlfigueiredo** [comment]
React Ninja

**Comment by vitorsavedra** [comment]
React Ninja

**Comment by nubioknupp** [comment]
React Ninja

**Comment by hfmarques** [comment]
JavaScript Ninja

**Comment by Leodma** [comment]
Vou de Git Github .. a concorrência ta menor .. kkkkk

**Comment by julian-kuroiwa** [comment]
React Ninja

**Comment by gmanaclaudia** [comment]
React Ninja :)

**Comment by danielmmellop** [comment]
React Ninja

**Comment by gustavofsantos** [comment]
React Ninja

**Comment by emerson-franca** [comment]
Git Github =)

**Comment by RuanAyram** [comment]
React Ninja

**Comment by josuetex** [comment]
JavaScript Ninja

**Comment by eynan** [comment]
React Ninja

**Comment by joeyclapton** [comment]
React Ninja

**Comment by rafaelpradella** [comment]
JavaScript Ninja

**Comment by linusdan** [comment]
REACT NINJA!!!
QUERO E MUITO!!!
Se eu ganhar, vai ser um presentão de natal!!!

**Comment by rafaelangical** [comment]
React Ninja

**Comment by marcelomarsola** [comment]
JavaScript Ninja !

**Comment by jessielyoliveira** [comment]
React Ninja

**Comment by MoesioMarcelino** [comment]
React Ninja

**Comment by felipe-pita** [comment]
React Ninja

**Comment by alexandreramosdev** [comment]
Javascript Ninja :tada:

**Comment by youphenrique** [comment]
React Ninja

**Comment by layshidani** [comment]
JavaScript ninja.
:)

**Comment by ViniciusGularte** [comment]
React Ninja

**Comment by xbrunosousa** [comment]
React Ninja =D

**Comment by leandroramos** [comment]
React Ninja!

**Comment by felipebalotim** [comment]
React Ninja

**Comment by robert-adam-dev** [comment]
Eu quero JavaScript ninja

**Comment by lesleyandrez** [comment]
React Ninja

**Comment by DanStos** [comment]
React Ninja xD

**Comment by vjordanmc** [comment]
JavaScript Ninja então. 🗡

**Comment by edneyroberto** [comment]
React Ninja :-)

**Comment by DanielleBK** [comment]
JavaScript ninja

**Comment by bkullmanns** [comment]
JavaScript Ninja

**Comment by jhonattan-santos** [comment]
React Ninja 😍

**Comment by viniciusdocanto** [comment]
JavaScript Ninja

**Comment by BrunoDamacena** [comment]
React Ninja

**Comment by luysla** [comment]
JavaScript Ninja

**Comment by raulfernando** [comment]
React Ninja

**Comment by mutheusalmeida** [comment]
JavaScript Ninja

**Comment by djhanlj** [comment]
JavaScript Ninja

**Comment by pauloilarde** [comment]
React ninja

**Comment by esdrascavalcs** [comment]
React Ninja

**Comment by franciscpd** [comment]
JavaScript Ninja :)

**Comment by Dougwan** [comment]
JavaScript Ninja <3

**Comment by viniciuscosta89** [comment]
Javascript Ninja 😃

**Comment by albjoaov** [comment]
React Ninja

**Comment by rodrigocode4** [comment]
React Ninja

**Comment by brazica** [comment]
JavaScript Ninja

**Comment by BrunoBeneducci** [comment]
JavaScript Ninja!

**Comment by meicle** [comment]
React Ninja!

**Comment by bonejah** [comment]
Javascript Ninja

**Comment by leonardoduarte1** [comment]
React Ninja

**Comment by jjazvdo** [comment]
_JavaScript Ninja_ **yaay**

**Comment by kavalcante** [comment]
React Ninja

**Comment by robertofilho1** [comment]
Javascript Ninja

**Comment by rodrigets** [comment]
JavaScript Ninja

**Comment by kelsonthony** [comment]
JavaScript Ninja!

**Comment by thicsilva** [comment]
React Ninja

**Comment by jilvanx** [comment]
JavaScript Ninja

**Comment by netojocelino** [comment]
JavaScript Ninja

**Comment by NathanReuter** [comment]
React Ninja

**Comment by waghcwb** [comment]
React Ninja :yum:

**Comment by faelribeiro22** [comment]
React Ninja

**Comment by f3rn4nd0** [comment]
React Ninja

**Comment by gabrielvf64** [comment]
JavaScript Ninja

**Comment by Carolis** [comment]
React Ninja ;)

**Comment by vituhernandez** [comment]
JavaScript Ninja

**Comment by caioincau** [comment]
React Ninja :D

**Comment by rafaelji** [comment]
React Ninja

**Comment by leoamrl** [comment]
Javascript Ninja

**Comment by wpbarcelos** [comment]
React Ninja

**Comment by taranttini** [comment]
(o sorteio não será por curso, e sim por pessoa, que vai ganhar o curso escolhido)
só porque eu ia escolher o *git  e git hub* por não ter envios =) ho ho ho esse papai noel poderia ser mais bondoso =)
se eu não ganhar eu gostaria ao menos das apostilas =) desse curso.

React Ninja

**Comment by caiobubb** [comment]
JavaScript Ninja

**Comment by ghost** [comment]
React Ninja

**Comment by borgestj** [comment]
JavaScript Ninja

**Comment by mateusmiguel** [comment]
JavaScript Ninja

**Comment by vargasgabriel** [comment]
JavaScript Ninja

**Comment by Haaragard** [comment]
JavaScript Ninja

**Comment by vNNi** [comment]
JavaScript Ninja

**Comment by Gpaula95** [comment]
React Ninja !

**Comment by romulo2735** [comment]
JavaScript Ninja :100:

**Comment by ivosoares** [comment]
React Ninja

**Comment by CristhyanKo** [comment]
React Ninja

**Comment by andersoonweb** [comment]
React ninja <3

**Comment by rsip22** [comment]
React Ninja

Por favor, sorte, nunca te pedi nada

**Comment by mauriciomutte** [comment]
React Ninja ❤️

**Comment by ipetinate** [comment]
JavaScript Ninja

**Comment by fredmosc** [comment]
React Ninja

**Comment by RicardoWEBSiTE** [comment]
React Ninja

Em qui, 13 de dez de 2018 às 16:59, Frederick Moschkowich <
notifications@github.com> escreveu:

**Comment by danielnunesdc** [comment]
React Ninja

**Comment by marianadasilvadev** [comment]
React Ninja!

**Comment by caiozf** [comment]
React Ninja

**Comment by luizcencio** [comment]
JavaScript Ninja

**Comment by emerson-paiva** [comment]
React Ninja

**Comment by fabianomaximiano** [comment]
Javascript

**Comment by gagres** [comment]
JavaScript Ninja

**Comment by vitorfreitas** [comment]
React ninja!

**Comment by Henrique018** [comment]
React Ninja :D

**Comment by DEXTERNATAN** [comment]
React  ninja

**Comment by HmmerHead** [comment]
JavaScript Ninja

**Comment by nantius** [comment]
React Ninja

**Comment by kscastro** [comment]
React Ninja

**Comment by felipemoraesdesign** [comment]
React Ninja

**Comment by igorthurow** [comment]
React Ninja

**Comment by linconkusunoki** [comment]
JavaScript Ninja

**Comment by vinigr** [comment]
React Ninja

**Comment by silvaMatheus** [comment]
React Ninja

**Comment by silvaMatheus** [comment]
React Ninja

**Comment by FilhoSergio** [comment]
JavaScript Ninja

**Comment by danielnunesnoc** [comment]
JavaScript Ninja

**Comment by tiagonevestia** [comment]
React Ninja

**Comment by gjrmacedo** [comment]
React Ninja

**Comment by helenapaixao** [comment]
JavaScript Ninja

**Comment by betiol** [comment]
React Ninja

**Comment by juniorstreichan** [comment]
React Ninja

**Comment by brunoravanhani** [comment]
React Ninja

**Comment by pietrobelluno** [comment]
React Ninja

**Comment by samuelrvg** [comment]
Git e Github

**Comment by ghost** [comment]
Github ninja

**Comment by ulisseslimaa** [comment]
JavaScript Ninja

**Comment by adamsalves** [comment]
Git e Github

**Comment by gabriel-hahn** [comment]
Git e Github

Já tenho os outros dois hehe

**Comment by rfcaio** [comment]
JavaScript Ninja.

**Comment by douglasluis** [comment]
React Ninja.

**Comment by douglas-neves** [comment]
JavaScript Ninja

**Comment by marqueslu** [comment]
React Ninja

**Comment by williambauch** [comment]
React Ninja

**Comment by rafahsena** [comment]
React Ninja <3

**Comment by ptkpereira** [comment]
JavaScript Ninja

**Comment by cenoura** [comment]
JavaScript Ninja :smile:

**Comment by geovanerocha** [comment]
Javascript Ninja

**Comment by DazosDev** [comment]
React Ninja

**Comment by maryEllTest** [comment]
React Ninja

**Comment by kojjee** [comment]
React Ninja

**Comment by danilogila** [comment]
React Ninja

**Comment by Shadaw** [comment]
React Ninja

**Comment by fesanto** [comment]
Git e Github

**Comment by tiagocalisto** [comment]
Será um ótimo presente para quem ganhar.

**Comment by mellmiranda-zz** [comment]
React ninja

**Comment by natashamorais** [comment]
React ninja :)

**Comment by williamxsp** [comment]
JavaScript Ninja :D

**Comment by wps13** [comment]
React Ninja

**Comment by NicholasKuchiniski** [comment]
JavaScript Ninja

**Comment by Marilzon** [comment]
JavaScript Ninja

**Comment by juliolpiva** [comment]
JavaScript Ninja

**Comment by ghost** [comment]
React Ninja

**Comment by Hoffmannn** [comment]
React Ninja

**Comment by gabrielafoliveira** [comment]
React Ninja =D

**Comment by brazieljr** [comment]
React Ninja!!!!

**Comment by pissulin** [comment]
React Ninja

**Comment by miayuyo** [comment]
React Ninja :)

**Comment by albertosveloso** [comment]
React Ninja

**Comment by munizart** [comment]
JavaScript Ninja
Feliz Natal :D :fireworks:

**Comment by rap0so** [comment]
JavaScript Ninja
:laughing:

**Comment by leonardopavanrocha** [comment]
React Ninja!
Feliz natal :tada:

**Comment by oliveira** [comment]
React Ninja!

**Comment by gmreis** [comment]
React Ninja

**Comment by diogens** [comment]
React Ninja

**Comment by karencosta** [comment]
React Ninja

**Comment by allanco91** [comment]
JavaScript Ninja

**Comment by higorbarbossa** [comment]
JavaScript Ninja

**Comment by cleisoncarlos** [comment]
JavaScript Ninja 🎄 🎅 🎁

**Comment by wilsonafs** [comment]
Git e Github

**Comment by JacksonOsvaldo** [comment]
JavaScript Ninja

**Comment by ChrislaRodrigues** [comment]
Git e Github

**Comment by gfmarcelo** [comment]
JavaScript Ninja

**Comment by Roneycacau** [comment]
React Ninja

**Comment by flores-joice** [comment]
React Ninja

**Comment by caioreis123** [comment]
React Ninja

**Comment by msaidel** [comment]
React Ninja

**Comment by espudaro** [comment]
Git e Github

**Comment by LeoRedin** [comment]
Olha ai oh! React Ninja! 😎

**Comment by matheus-vieira** [comment]
React Ninja

**Comment by matheus-vieira** [comment]
JavaScript Ninja

**Comment by WandersonAlves** [comment]
React Ninja!

**Comment by filipeluiz** [comment]
JavaScript Ninja

**Comment by alabliuk** [comment]
JavaScript Ninja

**Comment by luizbalisa** [comment]
React Ninja!

**Comment by claubatista** [comment]
Javascrip Ninja

**Comment by ghost** [comment]
React Ninja! :)

**Comment by eliasmelgaco** [comment]
React Ninja

**Comment by NaldsonChagas** [comment]
React Ninja

**Comment by nocrag** [comment]
JavaScript Ninja

**Comment by mauromattos00** [comment]
React Ninja!

**Comment by veridiem** [comment]
React Ninja

**Comment by FerrazBruno** [comment]
JavaScript Ninja

**Comment by calixtojj** [comment]
React Ninja

**Comment by matheus-vieira** [comment]
Git e Github

**Comment by ghost** [comment]
React Ninja

**Comment by matheusboari** [comment]
Git e Github

**Comment by wschenkel** [comment]
React ninja!

**Comment by f0ntana** [comment]
React Ninja!

**Comment by axdemelas** [comment]
React ninja

**Comment by filipeev** [comment]
React Ninja

**Comment by ThiiagoSeven** [comment]
JavaScript Ninja :laughing:

**Comment by saranascimento** [comment]
React Ninja

**Comment by jampow** [comment]
javascript ninja

**Comment by ghost** [comment]
Javascript Ninja

**Comment by rafaelvie** [comment]
React ninja! Presenteia nois Felipão!

**Comment by ellendutra** [comment]
JavaScript Ninja

**Comment by pedroribeiro** [comment]
JS NINJA!!

**Comment by ghost** [comment]
JavaScript Ninja 😎⚔️

**Comment by bibixledo** [comment]
React Ninja 🙅

**Comment by HigorGomesLima** [comment]
React Ninja

**Comment by jorgepopel** [comment]
Também React Ninja

**Comment by mailysantos** [comment]
React Ninja

**Comment by montanhes** [comment]
React Ninja

**Comment by GustavoLima93** [comment]
React Ninja

**Comment by genivaldosilva** [comment]
React Ninja 🤘

**Comment by DiegoRamires** [comment]
React Ninja

**Comment by justlikedev** [comment]
JavaScript Ninja

**Comment by MarceloWis** [comment]
React Ninja

**Comment by oliveiraanthony** [comment]
JavaScript

**Comment by AgostinhoBarbosa** [comment]
react ninja

**Comment by GesielFreitas** [comment]
React Ninja

**Comment by renanshin** [comment]
React Ninja

**Comment by sidneisimeao** [comment]
Javascript Ninja

**Comment by MichaelAlbertoni** [comment]
JavaScript ninja 😎

**Comment by lucas-stellet** [comment]
React Ninja

**Comment by dcerbolinha** [comment]
React Ninja :)

**Comment by isamuhiro** [comment]
Javascript Ninja 💅

**Comment by demellon** [comment]
JavaScript Ninja

**Comment by CassyJones** [comment]
React Ninja. 😎

**Comment by PdziM** [comment]
React Ninja 😏

**Comment by dmaiabjj** [comment]
JavaScript Ninja

**Comment by cirocfc** [comment]
React Ninja :turtle: :crossed_swords:

**Comment by Giovani-Silva** [comment]
**React Ninja**

**Comment by PabloFerreiraB** [comment]
### Eu quero ;)
# `` JavaScript Ninja `` 
## Feliz natal à todos!

**Comment by ferpavanello** [comment]
React Ninja

**Comment by leogreal** [comment]
# React Ninja

**Comment by ciroricardo** [comment]
React Ninja

**Comment by Aurenilson10** [comment]
Eu quero Javascript Ninja

**Comment by brunapmagalhaes** [comment]
JavaScript Ninja

**Comment by gregolly** [comment]
React Ninja

**Comment by d-rocha** [comment]
React Ninja please!

**Comment by lemamon** [comment]
JavaScript Ninja

**Comment by GuiMoraesDev** [comment]
JavaScript Ninja

**Comment by vitorgiovane** [comment]
**React Ninja**

**Comment by mrsventura** [comment]
React Ninja ;)

**Comment by Dougwan** [comment]
JavaScprit Niiiiinjaaaaa

**Comment by JoaoBKS** [comment]
React Ninja

**Comment by mocraveirodev** [comment]
Opaaaa... quero mt o React Ninja! <3 Os outros sao mt bons!!!

**Comment by helderberto** [comment]
Git e Github 💯

**Comment by lrnbtt** [comment]
Javascript Ninja

**Comment by larafelix** [comment]
JavaScript Ninja

**Comment by marcoaugustoandrade** [comment]
JavaScript Ninja

**Comment by icarobuzz** [comment]
Git Ninja

**Comment by rafavschneider** [comment]
JavaScript Ninja

**Comment by iamPedroVictor** [comment]
React Ninja.

**Comment by dev-alves** [comment]
React Ninja! ♥️:up:

**Comment by nonatodiego** [comment]
React Ninja!

**Comment by epontes** [comment]
JavaScript Ninja

**Comment by anndreyy** [comment]
JavaScript Ninja =D

**Comment by laeldev** [comment]
React Ninja ;)

**Comment by TDamiao** [comment]
JavaScript Ninja

**Comment by mateusdeleon** [comment]
Git e GitHub.

**Comment by gbdsantos** [comment]
**React Ninja**

**Comment by DeividVeloso** [comment]
React Ninja

**Comment by carlosdouglas1122** [comment]
JavaScript Ninja

**Comment by lidymonteirowm** [comment]
React Ninja!

**Comment by idaniloreis** [comment]
JavaScript Ninja

**Comment by CleysonPH** [comment]
Se por um acaso do destino eu for sorteado eu quero ganhar o curso React Ninja 😉

**Comment by devrafacarvalho** [comment]
JavaScript Ninja

**Comment by felipesaban** [comment]
React Ninja!

**Comment by JaderPS** [comment]
JavaScript Ninja

**Comment by ViniciusFerreiraa** [comment]
React Ninja

**Comment by mandado** [comment]
React Ninja

**Comment by andrelsa** [comment]
JavaScript Ninja

**Comment by Autobele** [comment]
Eu sendo o felizardo, desejo o **React Ninja**!

**Comment by miqueiaspenha** [comment]
React Ninja

**Comment by rafaelmontarroyos** [comment]
React Ninja

**Comment by carlosevandro** [comment]
__React Ninja__. 

## Feliz Natal e Feliz Ano Novo para todos. ##

**Comment by christophermoura** [comment]
Gostei da iniciativa, "Papai @fdaciuk Noel"! 🎅 

React Ninja

**Comment by camillodev** [comment]
React Ninja

**Comment by munizart** [comment]
@rodrigobranas rola um sorteio do masterclass também? xD

**Comment by luchiago** [comment]
<b>React Ninja<b>

**Comment by brognilucas** [comment]
O React Ninja já tenho, quero o Javascript Ninja >_<'

**Comment by rivanildojr** [comment]
React Ninja

**Comment by felipemv13** [comment]
React Ninja

**Comment by znobrega** [comment]
React Ninja

**Comment by samuelematias** [comment]
React Ninja 🤩

**Comment by glaufercharao** [comment]
React Ninja

**Comment by kauanmocelin** [comment]
JavaScript Ninja

**Comment by ln0rd** [comment]
React Ninja

**Comment by taraujo310** [comment]
React Ninja

**Comment by paulhdev** [comment]
React Ninja

**Comment by mayckxavier** [comment]
React Ninja

**Comment by isapires** [comment]
JavaScript Ninja

**Comment by luisrabock** [comment]
React Ninja

**Comment by eriksid** [comment]
React Ninja

**Comment by osterloh** [comment]
JavaScript Ninja

**Comment by rvsantos** [comment]
React Ninja

**Comment by isacmoura** [comment]
React Ninja

**Comment by joaomsneto** [comment]
React Ninja

**Comment by josemalcher** [comment]
React Ninja

**Comment by sergiobarbosaweb** [comment]
React Ninja

**Comment by rodineicosta** [comment]
React Ninja

**Comment by renzopalmieri** [comment]
JavaScript ninja

**Comment by gabriel-barreto** [comment]
JavaScript Ninja

**Comment by Joaovrog** [comment]
JavaScript ninja!

**Comment by nandomoreirame** [comment]
Git e Github

**Comment by davidalves1** [comment]
React Ninja! s2

**Comment by msmosso** [comment]
React Ninja :+1:

**Comment by gabrieladaltoe** [comment]
React Ninja!

**Comment by ZankoyDll1999** [comment]
React Ninja!

**Comment by CesarDenis** [comment]
React Ninja

**Comment by EmanoelLopes** [comment]
JavaScript Ninja 🚀

**Comment by m2rcyo** [comment]
JavaScript Ninja

**Comment by ericksoncst** [comment]
Javascript Ninja

**Comment by fdaciuk** [comment]
As participações estão encerradas! Obrigado a todos que participaram =)

Daqui a pouco vamos fazer o sorteio ao vivo pelo YouTube. Aguardem atualizações nesse mesmo post, e boa sorte a todos :D

**Comment by fdaciuk** [comment]
Fala povo! Quem quiser acompanhar o sorteio ao vivo, segue esse link:
https://www.youtube.com/watch?v=KKYbd_JI_LA

Vai ser às 18h! :D

**Comment by fdaciuk** [comment]
**Parabéns aos ganhadores:** :dancer: :dancer: :sparkles: 

- @pissulin (React Ninja)
- @Kamilahsantos (React Ninja)
- @eliasmelgaco (React Ninja)

Segunda-feira pela manhã já vai ter um comentário meu aqui mostrando como os ganhadores deverão proceder para resgatar os cursos free!

E pra quem não ganhou, não se preocupe! Vai ter cupom especial pra vocês :D

Fiquem ligados nessa issue que até segunda-feira teremos novidades :D 

**EDIT:** Boas festas a todos! :santa: :santa: 
Até quinta (27/12) eu resolvo isso, fiquem tranquilos! :D

**Comment by fdaciuk** [comment]
Fala povo! Agora sim trago novidades \o/

Para quem ganhou os cursos, você precisa fazer o seguinte:
- Acesse o endereço: https://queroser.ninja/sorteio
- Faça login com seu GitHub clicando no botão que aparece no meio da tela;
- Pronto! Você vai receber um link com seu cupom :D 

Se você participou, mas não ganhou, pode fazer o mesmo procedimento que tem um cupom de desconto especial lá :D 

Obrigado mais uma vez a todos que participaram, e ano que vem tem muito mais \o/

**Comment by osterloh** [comment]
JavaScript Ninja

**Comment by rafaelvie** [comment]
React ninja!

**Comment by rfcaio** [comment]
React Ninja!

**Comment by willianjusten** [comment]
Povo, essa discussão é de **2 anos atrás**, mas bom, vai que o @fdaciuk aceita fazer de novo? ahahhaha

**Reply by fdaciuk** [reply]
Agora que eu vi que dá pra responder por aqui kkk! ❤️

**Comment by fdaciuk** [comment]
Já que tu reviveu o post, vamos fazer ué hahah! ❤️ 
Mas vou abrir uma nova discussão pra fazer um novo sorteio, senão todo mundo aqui vai participar de novo haha :D
